### PR TITLE
Microk8s homepage refresh 19

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Created new index page with Microk8s from  [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#) and [design spec](https://github.com/ubuntudesign/web-squad/issues/915)
## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: [http://0.0.0.0:8027/](http://0.0.0.0:8027/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Make sure the MicroK8s matches the  [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#) and [design spec](https://github.com/ubuntudesign/web-squad/issues/915

## Issue / Card

Fixes https://github.com/canonical-websites/microk8s.io/issues/25
## Screenshots

image

image

image

### CODE REVIEW

**Comment by kwm14** [comment]
## Visual update
Adding to @Greggless review above 😄 

### Hero
- [ ] background-image: linear-gradient(-121deg, #000000 0%, #111111 51%, #262626 100%);
- [ ] suru folds - https://assets.ubuntu.com/v1/f9c6909a-microk8s-suru-right.png

### Desktop
takeover-large-screen-004

### Mobile
takeover-small-screen-004

**Comment by inuwa** [comment]
@kwm14 All done thanks for the help.

File: index.html [hunk 1]
```diff
@@ -1,227 +1,341 @@
...
+            <a href="/">Overview</a>
+          </li>
+          <li class="p-navigation__link" role="menuitem">
+            <a href="https://github.com/ubuntu/microk8s/blob/master/README.md">Docs</a>
+          </li>
+          <li class="p-navigation__link" role="menuitem">
+            <a class="p-link--external" href="https://discuss.kubernetes.io/tags/microk8s">Discourse</a>
+          </li>
+          <li class="p-navigation__link" role="menuitem">
+            <a class="p-link--external" href="https://github.com/ubuntu/microk8s/issues">Code</a>
```
Comment (Greggless) [inline index.html]:
```suggestion
            <a class="p-link--external" href="https://github.com/ubuntu/microk8s">Code</a>
```

Comment (Greggless) [reply index.html]: My bad, sorry

File: index.html [hunk 2]
```diff
@@ -1,227 +1,341 @@
...
-          </button>
-          <a class="navbar-brand" href="#"><b>microk8s</b></a>
-        </div>
-        <div class="navbar-collapse collapse">
-          <ul class="nav navbar-nav navbar-right">
-            <li><a href="https://github.com/ubuntu/microk8s">Github</a></li>
-          </ul>
+  <main id="main-content">
+    <section class="p-microk8s-hero">
+      <div class="p-microk8s-suru p-strip is-dark is-bordered">
```
Comment (kwm14) [inline index.html]: Remove `is-bordered` class as it's not needed on the hero strip 👍

File: index.html [hunk 3]
```diff
@@ -1,227 +1,341 @@
...
+            <h4>A single package of k8s that installs on <s>41</s> 42 flavours of Linux. Made for developers and great for
+              appliances.</h4>
+            <div class="p-logo-snap-store">
+              <a href="https://snapcraft.io/microk8s" title="Get it from the Snap Store">
+                <img class="p-logo-snap-store__image" src="https://assets.ubuntu.com/v1/fa3c60f8-snap-store-white.svg"
+                  alt="Snap Store" />
+              </a>
+            </div>
+          </div>
+          <div class="col-4 u-align--center u-vertically--center p-logo">
```
Comment (kwm14) [inline index.html]:
```suggestion
          <div class="col-4 u-align--center u-vertically-center p-logo">
```

Comment (kwm14) [reply index.html]: This is why we couldn't get it to work 😄

**Pull request merged**